cms/templates/studio_xblock_wrapper.html

```diff
@@ -190,7 +192,7 @@
     </div>
     % if not is_root:
         <div class="wrapper-xblock-message xblock-validation-messages" data-locator="${xblock.location}"/>
-        % if xblock_url:
+        % if xblock_url and not has_not_configured_message:
```

Comment:
Will this have impacts on other xblocks? curious if other have has_not_configured_message

Reply:
I was also worried about the same thing, but I couldn't find anything that used it from the basic components that I looked at.